guides/source/7_1_release_notes.md

```diff
@@ -484,12 +484,14 @@ Please refer to the [Changelog][action-pack] for detailed changes.
 
 *   Deprecate `config.action_dispatch.return_only_request_media_type_on_content_type`.
 
-*   Deprecate `AbstractController::Helpers::MissingHelperError`
+*   Deprecate `AbstractController::Helpers::MissingHelperError`.
 
 *   Deprecate `ActionDispatch::IllegalStateError`.
 
 *   Deprecate `speaker`, `vibrate`, and `vr` permissions policy directives.
 
+*   Deprecate `true` and `false` values for `action_dispatch.show_exceptions` in favor of `:all`, `:rescuable`, or `:none`.
```

Comment:
```suggestion
*   Deprecate `true` and `false` values for `config.action_dispatch.show_exceptions` in favor of `:all`, `:rescuable`, or `:none`.
```

I see a few examples of other configuration values mentioned in the release notes that include `config.`, so I think we should also include it here for consistency

Reply:
Sounds good!